src/components/Terminal/index.tsx

```diff
@@ -0,0 +1,17 @@
+import RootStore from '../../stores/rootStore';
+import {connect} from '../connect';
+import Terminal from './Terminal';
+
+export interface TerminalProps {
+  rootStore: RootStore;
```

Comment:
i might be wrong but i can't see how the `Terminal` component will get the access to `rootStore` itself. Provider injects destructured `rootStore` as before

Reply:
RootStore instance is injected to the component, so we can use it's methods